pkg/collector/osmlogs_collector.go

```diff
@@ -97,6 +98,24 @@ func collectNamespaceMetadata(collector *OsmLogsCollector, namespaces []string,
 	return nil
 }
 
+// Collects services for each ns in a given mesh
+func collectNamespacesServices(collector *OsmLogsCollector, namespaces []string, rootPath, meshName string) error {
+	for _, namespace := range namespaces {
+		namespaceServicesFile := filepath.Join(rootPath, meshName+"_"+namespace+"_"+"services")
+		namespaceServices, err := utils.RunCommandOnContainer("kubectl", "get", "services", "-n", namespace)
```

Comment:
@johnsonshi I think we should add a selector like `app.kubernetes.io/name=openservicemesh.io` so that we only get osm's services in case the user adds their own services to the ns, wdyt?

Reply:
Also, should this also have the `-o` flag set to json so that we get the details of the service too?

Reply:
@shalier it may be better to include all services in a given osm mesh's namespace. That helps us understand the mesh environment better.